docs/sql_permissions.md

```diff
@@ -0,0 +1,82 @@
+# SQL Permissions
+
+Promscale adopts a Role Based Access Control (RBAC) philosophy to managing permissions
+at the SQL layer. Many users will choose to simply use a single PostgreSQL user for all
+Promscale operations (using a single owner user as defined belows).
+But we also allow users more granular control over permissions management.
+
+Promscale defines several roles:
+- **Owner** this is the owner of the Promscale schema and objects. It is defined
+  as the user that originally installed/migrated Promscale. This user needs to be
+  able to install several PostgreSQL extensions and thus either needs to be a
```

Comment:
Maybe just link to the whitelist discussion below rather than state a similar thing here that distracts?